doc/man3/SSL_CTX_set_tlsext_max_fragment_length.pod

```diff
@@ -0,0 +1,78 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_tlsext_max_fragment_length, SSL_set_tlsext_max_fragment_length, SSL_get_tlsext_max_fragment_length - enable Maximum Fragment Length Negotiation extension
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ long SSL_CTX_set_tlsext_max_fragment_length(SSL_CTX *ctx, long mode);
+ long SSL_set_tlsext_max_fragment_length(SSL *ssl, long mode);
+
+ long SSL_get_tlsext_max_fragment_length(SSL *ssl);
```

Comment:
There should probably be some discussion somewhere on the relationship between max_fragment_length, max_send_fragment and split_send_fragment. Possibly even put them all in the same man page.